tests/components/zwave_js/test_init.py

```diff
@@ -1125,7 +1125,7 @@ async def test_node_model_change(hass, zp3111, client, integration):
     motion_entity = "binary_sensor.4_in_1_sensor_home_security_motion_detection"
     state = hass.states.get(motion_entity)
     assert state
-    assert state.name == "4-in-1 Sensor: Home Security - Motion detection"
+    assert state.name == "4-in-1 Sensor Home Security - Motion detection"
```

Comment:
Shouldn't the entity name just be Motion detection?

Reply:
I don't know enough of the use cases to speak to it but that sounds reasonable